.gitignore

```diff
@@ -278,3 +278,5 @@ Apidaze.SDK.Tests.Integration/UpdateExternalScriptUrl/client-secrets.json
 /Apidaze.SDK.Tests.Integration/SubApplicationExternalScriptsExample/client-secrets.json
 /Apidaze.SDK.Tests.Integration/ListApplications/client-secrets.json
 /Apidaze.SDK.Tests.Integration/GetApplicationActionExample/client-secrets.json
+Apidaze.SDK.Tests.Integration/UploadMediaFile/client-secrets.json
+Apidaze.SDK.Tests.Integration/DownloadMediaFile/client-secrets.json
```

Comment:
The other ones have a leading slash but these two don't. Is that intentional?

Reply:
Resolved.